nvme-print.c

```diff
@@ -767,10 +769,45 @@ void nvme_show_endurance_log(struct nvme_endurance_group_log *endurance_log,
 	nvme_print(endurance_log, flags, endurance_log, group_id, devname);
 }
 
-const char *nvme_degrees_string(long t, bool fahrenheit)
+static bool is_fahrenheit_country(const char *country)
+{
+	const char* fahrenheit_countries[] = {"AS", "AQ", "BS", "BZ", "KY", "CY", "GU", "LR", "MH",
+					      "FM", "MS", "MP", "PW", "PR", "KN", "TC", "US", "VG",
+					      "VI", NULL};
+	int i;
+
+	for (i = 0; fahrenheit_countries[i]; i++) {
+		if (!strcmp(country, fahrenheit_countries[i]) )
+			return true;
+	}
+
+	return false;
+}
+
+static bool is_temperature_fahrenheit(void)
+{
+	const char *locale;
+	char country[3];
+	char *underscore;
+
+	setlocale(LC_ALL, "");
+	locale = setlocale(LC_ALL, NULL);
+
+	underscore = strchr(locale, '_');
+	if (underscore != NULL && strlen(underscore) > 1)
+		locale += 3;
```

Comment:
Hmm, we might a bit more smarter here. IIRC, there are locales which have more than two chars before the underscore. Maybe `locale += strlen(understore)`?